helper_dart/pubspec.lock

```diff
@@ -641,4 +500,3 @@ packages:
     version: "3.1.1"
 sdks:
   dart: ">=2.17.5 <3.0.0"
-  flutter: ">=3.0.0"
```

Comment:
is this expected?

Reply:
Yes, it was here due to `shared_preferences` dependency, and this it the reason why it was replaced by `hive`. helper_dart is a pure Dart package and we don't want it to depend on Flutter.